test/integration/util/util.go

```diff
@@ -120,7 +127,7 @@ type OcRunner struct {
 }
 
 func NewOcRunner() *OcRunner {
```

Comment:
Maybe this constructor should actually take the path. This way it becomes more flexible. One can determine the right path from the outside and create a Runner with any odd oc version, not just the one provided by minishift.json